src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -24,6 +21,7 @@ internal abstract class PEAssemblyBuilderBase : PEModuleBuilder, Cci.IAssemblyRe
         private SynthesizedEmbeddedAttributeSymbol _lazyEmbeddedAttribute;
```

Comment:
>_lazyEmbeddedAttribute [](start = 51, length = 22)

Thinking out loud: Is there another way of selecting the correct constructor rather than index? what happens if later on another constructor was added?
Can we filter based on signature?